build/Targets/Imports.targets

```diff
@@ -118,6 +118,12 @@
     </When>
   </Choose>
 
+  <!-- Settings for localization -->
+  <PropertyGroup>
```

Comment:
Rather than create a new property group I would just add this to an existing group.